src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(number) {
+      calculator.result += number;
+    },
```

Comment:
Would be better if you will return results and save them in operate method
```suggestion
    add(num1, num2) {
      return num1 + num2;
    },
```